capa/features/extractors/ts/build.py

```diff
@@ -0,0 +1,10 @@
+from tree_sitter import Language
+
+build_dir = "build/my-languages.so"
+languages = [
+    "vendor/tree-sitter-c-sharp",
+]
+
+
+def ts_build():
+    Language.build_library(build_dir, languages)
```

Comment:
what does this do? can we install TS C# via pypi? if so it should be specified in `setup.py`.

Reply:
Tree-sitter requires that, in addition to pip-installing Tree-sitter, bindings for languages of interest are git cloned and built into a shared library using `Language.build_library(destination, source)`